usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/clients/MealManagerClient.java

```diff
@@ -0,0 +1,143 @@
+package org.pesmypetcare.usermanagerlib.clients;
+
+import com.google.gson.Gson;
+
+import org.json.JSONObject;
+import org.pesmypetcare.usermanagerlib.datacontainers.Date;
+import org.pesmypetcare.usermanagerlib.datacontainers.Meal;
+import org.pesmypetcare.usermanagerlib.datacontainers.MealData;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+
+public class MealManagerClient {
+    public static final String MEALNAME = "mealName";
+    public static final String KCAL = "kcal";
+    private static final String BASE_URL = "https://pes-my-pet-care.herokuapp.com/meal/";
+    private static final int POST = 0;
+    private static final int GET = 1;
+    private static final int DELETE = 2;
+    private static final int PUT = 3;
+    private static final Gson gson = new Gson();
+    private TaskManager taskManager;
+
+
+    /**
+     * Creates a meal eaten by a pet on the database.
+     * @param owner Username of the owner of the pet
+     * @param petName Name of the pet
+     * @param meal The meal entity that contains the attributes of the meal eaten by the pet
+     */
+    public void createMeal(String owner, String petName, Meal meal) {
+        JSONObject reqJson = meal.getBody().buildMealJson();
+        taskManager = new TaskManager();
+        taskManager.setTaskId(POST);
+        taskManager.setReqBody(reqJson);
+        taskManager.execute(BASE_URL + owner + "/" + petName + "/" + meal.getDate());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "/" appears 14 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5314325)